app/helpers/employers/employer_helper.rb

```diff
@@ -137,18 +137,18 @@ def render_plan_offerings(benefit_group, coverage_type)
       return "1 Plan Only" if benefit_group.single_plan_type?
       return "Sole Source Plan" if benefit_group.plan_option_kind == 'sole_source'
       if benefit_group.plan_option_kind == "single_carrier"
```

Comment:
We should look into a way of refactoring this file to remove state abbreviation references and make it more dynamic.